### ISSUE TITLE
Minesweeper Game

### ISSUE THREAD

**Issue by dreiekk** [Maveo/Spark#68]
### Description

There's already a library to generate those:
https://github.com/jozsefsallai/discord.js-minesweeper

### Example situation

Online Demo:
https://jozsefsallai.github.io/discord-minesweeper-generator/

**Comment by skillor** [comment]
It is not a game with multiple user engagement so I see no benefit for it being in Spark.

**Comment by skillor** [comment]
Do you mean each user can click once? Also is this pretty limited ( 5 fields width )

**Comment by dreiekk** [comment]
True, the size limit is quite small. Is it possible to send multiple messages, each containing a single row?
I guess everyone has unlimited moves, but only one at a time. After that, someone else has to advance the board.
